timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md

```diff
@@ -0,0 +1,385 @@
+
+* `tx_count`: Number of transactions per block
+* `block_fee_sat`: Transaction fee paid per block (in Sat)
+* `block_fee_usd`:  Transaction fee paid per block (in USD)
+* `stats_tx_fee_sat`: Stats for transaction fees
+* `avg_tx_size`: Average transaction size within the block
+* `avg_tx_weight`: Average transaction weight within the block
+* `block_size`: Total block size
+* `block_weight`: Total block weight
+* `max_tx_size`: Maximum transaction size within the block
+* `max_tx_weight`: Maximum transaction weight within the block
+* `min_tx_size`: Minimum transaction size within the block
+* `min_tx_weight`: Minimum transaction weight within the block
+
+Add a refresh policy to keep the continuous aggregate up-to-date:
+```sql
+SELECT add_continuous_aggregate_policy('one_hour_blocks',
+   start_offset => INTERVAL '3 hours',
+   end_offset => INTERVAL '1 hour',
+   schedule_interval => INTERVAL '1 hour');
+```
+
+
+## Continuous aggregate: coinbase transactions (miner revenue)
+Create a continuous aggregate called `one_hour_coinbase`. This view holds
+aggregated data about all the transactions that miners received as
+rewards each hour.
+
+
+```sql
+CREATE MATERIALIZED VIEW one_hour_coinbase
+WITH (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time) AS bucket,
+   count(*) AS tx_count,
+   stats_agg(output_total, output_total_usd) AS stats_miner_revenue,
+   min(output_total) AS min_miner_revenue,
+   max(output_total) AS max_miner_revenue
+FROM transactions
+WHERE is_coinbase IS TRUE
+GROUP BY bucket;
+```
+
+Running this query, you create these aggregate columns within the continuous
+aggregate:
+* `tx_count`: Number of coinbase transactions per day
+* `stats_miner_revenue`: Stats for miner revenue (both in Sat and USD)
+* `min_miner_revenue`: Minimum miner revenue received after mining a block per day
+* `max_miner_revenue`: Maximum miner revenue received after mining a block per day
+
+
+Add a refresh policy to keep the continuous aggregate up-to-date:
+```sql
+SELECT add_continuous_aggregate_policy('one_hour_coinbase',
+   start_offset => INTERVAL '3 hours',
+   end_offset => INTERVAL '1 hour',
+   schedule_interval => INTERVAL '1 hour');
+```
+
+In each continuous aggregate definition, the `time_bucket()` function controls
+how large the time buckets are. The examples all use 1-hour time
+buckets.
+
+## Generate insights with SQL
+Here are some questions you might ask about blockchain
+transactions, blocks, and miner revenue. For each question,
+you get a relevant SQL query and a chart that answers the question.
+
+**Questions**
+* Is there any connection between the number of transactions and the transaction fees?
+* Does the transaction volume affect the BTC-USD rate?
+* Do more transactions in a block mean the block is more expensive to mine?
+* What percentage of the average miner's revenue comes from fees vs. block rewards?
+* How does block weight affect miner fees?
+* What’s the average miner revenue per block?
+
```

Comment:
This feels unnecessary to me since we don't link to anything. Any other opinions?

Reply:
I can add anchor links so the reader can jump to the one they find the most interesting

Reply:
https://github.com/timescale/docs/pull/1176/commits/250b638967d8539d5237a70b48dd1a66c186ff20